aiida/restapi/resources.py

```diff
@@ -458,7 +458,7 @@ def get(self, id=None, page=None):  # pylint: disable=redefined-builtin,invalid-
 
                 if query_type == 'download' and download not in ['false', 'False', False] and results:
                     if results['download']['status'] == 200:
-                        data = results['download']['data']
+                        data = results['download']['data'].encode('utf-8')
```

Comment:
Can you explain why this change is necessary?

Didn't this work before?
Is `node._exportcontent` involved here?

Reply:
Thanks. For context, check "Additional context" on https://github.com/aiidateam/aiida-core/issues/5574.

I'm not sure how it worked before, as `node._exportcontent` says it always returns a binary string.

I'll do some debugging